docker-integrations/tendermint-python/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Use the official Python image as the base image
+FROM python:3.11
+
+# Set the working directory inside the container
+WORKDIR /app
+
+# Copy the Python code and protobuf files to the container
+COPY ./ ./
```

Comment:
If we create a Dockerfile which is inclusive of the files and dirs we don't want in the final image then this won't be copying the `.md` and other unneeded files.

Reply:
Updated `.gitignore` with the one proposed